client/src/app/user-boards/(realtime-canvas)/[boardId]/slides/[slideIndex]/_components/HorizontalMenu.tsx

```diff
@@ -29,50 +29,67 @@ const BoardHorizontalMenu: FC<HorizontalMenuProps> = ({ groupId, userCapabilitie
 
   return (
     <div className={`flex items-center justify-between border-b bg-background px-4 pl-0`}>
-      <div className="flex">
-        <Link
-          href={"/user-boards"}
-          className="h-min-[64px] flex h-[64px] w-[3.5em] cursor-pointer items-center justify-center border-r transition-colors hover:bg-muted"
-        >
-          <AppLogo width={20} height={20} />
-        </Link>
+      <div className="flex h-full">
+        <HoverCard openDelay={100} closeDelay={150}>
+          <HoverCardTrigger>
+            <Link
+              href={"/user-boards"}
+              className="h-min-[64px] flex h-[64px] cursor-pointer items-center justify-center gap-2 border-r p-4 transition-colors hover:bg-muted"
+            >
+              <AppLogo width={15} height={15} />
+              <span className={"text-sm font-semibold text-muted-foreground"}>Dashboard</span>
+            </Link>
+          </HoverCardTrigger>
+          <HoverCardContent>Go to dashboard</HoverCardContent>
```

Comment:
```suggestion
          <HoverCardContent>Go back to dashboard</HoverCardContent>
```
Or
```suggestion
          <HoverCardContent>Return to dashboard</HoverCardContent>
```

Reply:
sometimes, we can navigate to the board from board details view. In this case return has different context